cnxmltransforms/resolvers.py

```diff
@@ -412,14 +412,17 @@ def get_page_ident_hash(self, page_uuid, page_version,
         which may or may not include the book uuid depending on whether
         the page is within the book.
         """
+        from cnxepub import flatten_tree_to_ident_hashes  # XXX
```

Comment:
We already copied flatten_tree_to_ident_hashes from cnxepub to cnxmltransforms, so this line is not necessary.